components/compliance-service/api/tests/run.rb

```diff
@@ -100,7 +98,9 @@ def assert_grpc_error(details, code)
       assert_equal(details, error.details)
     end
 
-    assert_equal(code, error.code)
+    if code
+      assert_equal(code, error.code)
+    end
```

Comment:
This test isn't strictly necessary, but I've never been sure what these numbers represent and to find the correct value in the past I've just used 0 then seen what appears in the failure message.